assets/src/models/detour.ts

```diff
@@ -1,6 +1,25 @@
 import { LatLngLiteral } from "leaflet"
 import { ShapePoint, Stop } from "../schedule"
 import { CreateDetourMachineInput } from "./createDetourMachine"
+import { array, Infer, nullable, number, string, type } from "superstruct"
+
+export const SimpleDetour = type({
+  route: string(),
+  direction: string(),
+  name: string(),
+  intersection: string(),
+  updated_at: number(),
```

Comment:
I'm a little torn about this field. It really means different things in different contexts, so it wounds me a bit to have it just called `updated_at` (for active this is "active since", for instance)

For draft detours, `updated_at` means "last edited"
For active detours, `updated_at` means "active since"
For past detours, `updated_at` might mean "last used" (I asked a clarifying question about that - this might be more complicated)

The fact that the same field can be used for all three things feels like an implementation detail, and it feels to me like it's an implementation detail that really belongs in the backend.

But I also see merit in not over-complicating this at this stage.

Reply:
Hmm... At first, I felt the variations are presentational. `updated_at` should be taken at face-value on the backend and presented appropriately on the frontend.

On the other hand, I see how that meaning could get lost between backend => frontend, and so defining the meaning more explicitly could be valuable. Especially depending on how we implement being able to edit active detours down the road. Given we don't know yet how that will be presented on the frontend / what data model we'll need for detours at that point, I might be leaning towards the current simplicity?